### DISCUSSION TITLE
GE Logiq US device

### DISCUSSION THREAD

**Discussion by YousefMetwally** [PlusToolkit/PlusLib#1033] [Q&A]
Hello everyone, 
I wanted to know if I can use PLUStoolkit to stream the data from GE logiq US device? It is not mentioned in the supported devices so I wanted to ask if these are the only ones that it will work with? 
Thank you in advance

**Comment by ungi** [comment]
I think GE Logiq has no research interface to obtain images. If you must use this device and get real time images, you could use it's video output with a frame grabber (DVI to USB) that is supported by PLUS.
